### DISCUSSION TITLE
User Management

### DISCUSSION THREAD

**Discussion by mike14-2** [YOURLS/YOURLS#3740] [Plugins]
Hello,

I have three questions: 

1. I would like to know if it is possible to create multiple users and then manage them through a plug-in, or is this only possible via Config.php?

2. and then I would also like to know whether it is possible to see which user has created which abbreviation?

3. and I would also like to know whether it is possible for users to have different rights.

Thank you very much

**Comment by dgw** [comment]
The most comprehensive user management plugin I know of (from [our Awesome List](https://awesome.yourls.org/)) is https://github.com/joshp23/YOURLS-AuthMgrPlus

That said, which user created each link appears to be only used internally for filtering based on role permissions; the `user` is stored in the database, but admin table views don't appear to be modified.

I don't know of any plugin that allows user _creation_ via the web admin area. AuthMgrPlus still relies on defining users/roles in `config.php`.

YOURLS v2 (not yet public) has a much more robust user system. (@LeoColomb chime in here about future plans if you want 😸) However, there is no ETA for a new major release.
